server/requirements.txt

```diff
@@ -0,0 +1,7 @@
+Flask==1.1.1
+Flask-SQLAlchemy==2.4.1
+psycopg2==2.8.3
+flask-cors==3.0.8
+Flask-Migrate==2.5.2
+Flask-Testing==0.7.1
+Werkzeug==0.16.1
```

Comment:
There's linter support added for js. Maybe python should have linter/formatter too? 
https://pypi.org/project/autopep8/
https://pyre-check.org/